apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-allowed-users/facility-allowed-users.component.html

```diff
@@ -1,24 +1,53 @@
 <div>
   <h1 class="page-subtitle">{{'FACILITY_DETAIL.ALLOWED_USERS.TITLE' | translate}}</h1>
-  <perun-web-apps-refresh-button
-    (refresh)="refreshTable()">
-  </perun-web-apps-refresh-button>
-  <perun-web-apps-debounce-filter
-    [placeholder]="'FACILITY_DETAIL.ALLOWED_USERS.FILTER'"
-    (filter)="applyFilter($event)"></perun-web-apps-debounce-filter>
+  <div class="filters">
+    <perun-web-apps-refresh-button
+      class="mr-2"
+      (refresh)="updateTable = !updateTable">
+    </perun-web-apps-refresh-button>
+    <perun-web-apps-debounce-filter
+      class="mr-2 filter"
+      [placeholder]="'FACILITY_DETAIL.ALLOWED_USERS.FILTER'"
+      (filter)="applyFilter($event)"></perun-web-apps-debounce-filter>
+    <mat-slide-toggle
+      [(ngModel)]="allowed"
+      class="mr-2"
+      labelPosition="before">
+      {{'FACILITY_DETAIL.ALLOWED_USERS.FILTER_ALLOWED' | translate}}
+    </mat-slide-toggle>
```

Comment:
I think there needs to be better caption of what is going on
if you would do it with checkbox then this label would be enough with some tooltip - I dont think that users know what is the difference between allowed/assigned users
if you want to do this with the slider then I think we need the label on both side of it (just my opinion of course)